pkg/skaffold/deploy/resource/deployment.go

```diff
@@ -111,7 +114,7 @@ func (d *Deployment) IsStatusCheckComplete() bool {
 }
 
 func (d *Deployment) ReportSinceLastUpdated() string {
-	if d.status.reported {
+	if d.status.reported && !d.status.changed {
```

Comment:
This doesn't seem right as `.reported` is only ever changed here and it's always set to `true`.

I think you just want to track the last-reported status and the current status, and compare them here.

Reply:
@briandealwis This works correctly because, for every new status seen, we create a status with constructor `newStatus`. 
The `.reported` set to false. 
See deployment.go L57

Reply:
Ah ok 👍